cmd/dist/README.md

```diff
@@ -0,0 +1,11 @@
+<!-- README.md auto-generated by misc/genreadme; DO NOT EDIT. (or remove this line) -->
+
+# dist
+
+The dist command builds Tailscale release packages for distribution.
+
+To build signed packages for distribution (which only the build servers should
+be doing), pass --win-signer, or set the SIGNER environment variable.
```

Comment:
--win-signer is only for Windows or just a bad flag name?

Reply:
Deleted the readme, since it's not really contributing anything (and was outdated even before I copied it, now I look at what it said)